frontend/src/components/Likes.js

```diff
@@ -1,18 +1,21 @@
 import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
 import { handleLike, checkHasLiked } from "../utils";
 
 const Likes = props => {
   const { likesCount, id, likes } = props.post;
+  const userId = useSelector(state => {
+    if (!state.currentUser) return null;
+    else return state.currentUser.uuid;
+  });
```

Comment:
It's still more readable :-)
```suggestion
  const userId = useSelector(state => {
    if (!state.currentUser) {
      return null;
    } else {
      return state.currentUser.uuid;
    }
  });
```